### PR TITLE
Improve text: 'you have now' -> 'you now have'

### PR DESCRIPTION
**Pull request by leonklingele-work**
Really? Having to adjust 11 files for such a small change? 😿

### CODE REVIEW

**Comment by leonklingele-work** [comment]
Updated to include 'Signed-off-by' label.

**Comment by leonklingele-work** [comment]
> will increase coverage by 6.35%

wtf, why?

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
Well half of the files are tests for the system where this one template was duplicated....

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
@leonklingele-work Mind to open a PR against stable12 to bring those changes to an maintenance release?

**Comment by danopz** [comment]
Why is it `you now have an Nextcloud account` - shouldn't `now you have a Nextcloud account` be the correct phrase?

**Comment by leonklingele-work** [comment]
I prefer the one starting with `you`, feels more "personal". But yes, it should be `a` instead of `an` 😮

File: settings/Mailer/NewUserMailHelper.php [hunk 1]
```diff
@@ -130,7 +130,7 @@ public function generateTemplate(IUser $user, $generatePasswordResetToken = fals
 		} else {
 			$emailTemplate->addHeading($this->l10n->t('Welcome aboard %s', [$displayName]));
 		}
-		$emailTemplate->addBodyText($this->l10n->t('You have now an %s account, you can add, protect, and share your data.', [$this->themingDefaults->getName()]));
+		$emailTemplate->addBodyText($this->l10n->t('You now have an %s account, you can add, protect, and share your data.', [$this->themingDefaults->getName()]));
```
Comment (nickvergessen) [inline settings/Mailer/NewUserMailHelper.php]: The problem is we don't know whether the product name we insert here starts with a vocal or not, so not sure if an or a is correct.... the fun of translations or placeholders

Comment (jancborchardt) [reply settings/Mailer/NewUserMailHelper.php]: We could simply avoid it by rewording:
"Welcome to your Nextcloud account! …"

What do you think @jospoortvliet @leonklingele-work?

Comment (leonklingele-work) [reply settings/Mailer/NewUserMailHelper.php]: Much better.

Comment (MorrisJobke) [reply settings/Mailer/NewUserMailHelper.php]: @leonklingele-work Could you update the text on master first and then add it to the backport PR as well?

**Comment by MorrisJobke** [comment]
@leonklingele-work Just one further explanation of our workflow: Once a backport PR is opened we remove the label from the original PR. Sticking to that workflow, we are able to see if backports are still pending by going through all PRs that have this label assigned. 😉

**Pull request merged**